solidity/contracts/bridge/BridgeState.sol

```diff
@@ -444,19 +461,26 @@ library BridgeState {
     ///        it determines the percentage of the notifier reward from the
     ///        staking contact the notifier of a moving funds timeout receives.
     ///        The value must be in the range [0, 100]
+    /// @param _movedFundsMergeTxMaxTotalFee New value of the moved funds merge
+    ///        transaction max total fee in satoshis. It is the maximum amount
+    ///        of the total BTC transaction fee that is acceptable in a single
+    ///        moved funds merge transaction. This is a _total_ max fee for the
+    ///        entire moved funds merge transaction.
     /// @dev Requirements:
     ///      - Moving funds transaction max total fee must be greater than zero
     ///      - Moving funds dust threshold must be greater than zero
     ///      - Moving funds timeout must be greater than zero
     ///      - Moving funds timeout notifier reward multiplier must be in the
     ///        range [0, 100]
+    ///      - Moved funds merge transaction max total fee must be greater than zero
     function updateMovingFundsParameters(
         Storage storage self,
         uint64 _movingFundsTxMaxTotalFee,
         uint64 _movingFundsDustThreshold,
         uint32 _movingFundsTimeout,
         uint96 _movingFundsTimeoutSlashingAmount,
-        uint256 _movingFundsTimeoutNotifierRewardMultiplier
+        uint256 _movingFundsTimeoutNotifierRewardMultiplier,
+        uint64 _movedFundsMergeTxMaxTotalFee
```

Comment:
According to [a comment in another PR](https://github.com/keep-network/tbtc-v2/pull/236#discussion_r863557924) it would be good if we could follow a convention that `*Timeout`, `*SlashingAmount` and `*RewardMultiplier` parameters are always last in the given parameter group.

Therefore I would move `movedFundsMergeTxMaxTotalFee` ahead of the three mentioned parameters in the `updateMovingFundsParameters` function, `MovingFundsParametersUpdated` event, `BridgeState` parameter declarations, unit test constants. I would also move:
```
        require(
             _movedFundsMergeTxMaxTotalFee > 0,
             "Moved funds merge transaction max total fee must be greater than zero"
         );
  ```
  and
  ```
  - Moved funds merge transaction max total fee must be greater than zero
  ```
  up accordingly.
  I would also move `"when new moved funds merge transaction max total fee is zero" ` testcase up so that the order of tests for parameters is preserved.

Reply:
This is a "subgroup" of the "moving funds" group. There will be more parameters also related to slashing that will be put at the end. Those will be: `movedFundsMergeTimeout`, `movedFundsMergeTimeoutSlashingAmount` and `movedFundsMergeTimeoutNotifierRewardMultiplier`. I don't think they should have a separate group as they are tightly coupled with "moving funds".

Reply:
OK, that makes sense.